wireapp/wire-android
# chore: hide delete and share logs buttons

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

The app is crashing when the user clicks on delete logs while logging is disabled.

`com.wire.android.util.LogFileWriter.clearActiveLoggingFileContent
java.io.FileNotFoundException`

### Causes (Optional)

We are trying to delete something that is not yet created.

### Solutions

Hide Delete logs and share logs options when logging is off.


Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

Using a prod version, go to setting  --> Debug settings
You should not see those two options. if Enable logging is diabled


----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
